mrqacheck/audit.py

```diff
@@ -326,6 +322,15 @@ def process_scan(scan_dir, config_path, output_dir):
 
 # Main function
 def main():
+
+    # Grab the paths
+    SCAN_DIRECTORY = "/Volumes/Hera/Raw/MRprojects/Habit/"
+    filedir = Path(os.path.dirname(__file__))
+    LOG_FILE_PATH = filedir / "log_file.txt"
+    CONFIG_PATH = filedir / "mri-config.json"
+    OUTPUT_DIR = filedir / "output.txt"
+    # TODO: use argparser to grab theses settings from command line
+
```

Comment:
this should make all the files relative to the input file (and no longer hard coded to a specific user).
using the special `__file__` could be a problem later if this is ever made into a package. but by then argparser could take arguments from the command line